src/test/java/echoserver/ServerSocketWrapperTest.java

```diff
@@ -0,0 +1,58 @@
+package echoserver;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ServerSocketWrapperTest {
```

Comment:
What class are you actually testing here?

Reply:
Good point 👍 These tests should be in `EchoServerTest`